pyproject.toml

```diff
@@ -266,6 +266,7 @@ dl = [
   'FrEIA; python_version < "3.12"',
   'tensorflow<=2.14,>=2; python_version < "3.12"',
   'torch; python_version < "3.12"',
+  'transformers[torch]; python_version < "3.12"',
```

Comment:
According to this, it seems even 3.11 is not supported.

https://github.com/huggingface/transformers/blob/main/setup.py#L460

This may not have been updated though, I am not sure.

Reply:
That is weird... At least locally I was able to install it with python 3.11 and in the CI it is installed too:

https://github.com/sktime/sktime/actions/runs/8516512860/job/23325741934?pr=5796

Reply:
Yes, I verified this over weekend as well for a personal project. Don't know why they haven't updated, but probably we can ignore.